tools/lidar2image/main.cpp

```diff
@@ -0,0 +1,79 @@
+#include <Eigen/Core>
+#include <vector>
+#include <pcl/io/pcd_io.h>
+#include <pcl/point_types.h>
+#include <pcl/common/transforms.h>
+#include <opencv2/opencv.hpp>
+
+using namespace pcl;
+using namespace std;
+using namespace Eigen;
+
+
+int main() {
+    Eigen::Matrix4d cam_to_imu;
+    cam_to_imu.setIdentity();
+    cam_to_imu.block<3,3>(0,0)<< 0.9999930,  0.0013607,  0.0034907,
+            -0.0034799, -0.0078487,  0.9999632,
+            0.0013880, -0.9999683, -0.0078439;
+
+    cam_to_imu.block<3,1>(0,3)<<-0.1,0.11,0.11;
+
+    Eigen::Matrix4d imu_to_cam;
+    imu_to_cam.setIdentity();
+    imu_to_cam.block<3,3>(0,0)=cam_to_imu.block<3,3>(0,0).transpose();
+    imu_to_cam.block<3,1>(0,3)=-cam_to_imu.block<3,3>(0,0).transpose()*cam_to_imu.block<3,1>(0,3);
+
+    Eigen::Matrix4d lidar_to_imu;
+    lidar_to_imu<<1, 0, 0, 0,
+            0, 1, 0, 0,
+            0, 0, 1, 0.28,
+            0, 0, 0, 1;
+
+    Eigen::Matrix<double,3,4> P;
+    P<<264.9425, 0.0, 334.3975, 0.0,
+            0.0, 264.79, 183.162, 0.0,
+            0.0, 0.0, 1.0, 0.0;
+
+    int W=672;
+    int H=376;
+
+    cv::Mat image=cv::imread("3.png",cv::IMREAD_COLOR);
```

Comment:
the link cannot be used if I start the binary inside the build. 
```suggestion
    cv::Mat image=cv::imread("../data/3.png",cv::IMREAD_COLOR);
```